packages/daimo-contract/src/chain.ts

```diff
@@ -265,6 +268,17 @@ export const worldchain: DAv2Chain = {
   wrappedNativeToken: worldchainWETH,
 };
 
+export const blast: DAv2Chain = {
+  chainId: 81457,
+  name: "blast",
+  shortName: "blast",
+  cctpDomain: -1,
+  bridgeCoin: blastWETH,
+  localUSDC: blastUSDB, // Blast doesn't have local USDC
```

Comment:
definitely add a comment in the `DAv2Chain` definition that `localUSDC` is not always USDC

do we expose DAv2Chain in our APIs anywhere? if not, we can rename to `localUsdToken` or similar